src/AxeWindows.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28307.757
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29215.179
```

Comment:
Did we decide to embrace VS2019?

Reply:
I thought we had, but the docs still say 2017. Reverting...

---
In reply to: [325417068](https://github.com/microsoft/axe-windows/pull/149#discussion_r325417068) [](ancestors = 325417068)